cron.yaml

```diff
@@ -17,3 +17,6 @@ cron:
 - description: Send reminders to verify the accuracy of feature data.
   url: /cron/send_accuracy_notifications
   schedule: every monday 09:00
+- description: Copy over deprecated standardization field
+  url: /cron/write_standard_maturity
+  schedule: every day 09:00
```

Comment:
Since we will run this manually by pressing the button on the Cloud Console, how about a date that will not match?  E.g.,
schedule: 1 of jan 00:00
See https://cloud.google.com/appengine/docs/standard/python3/scheduling-jobs-with-cron-yaml#formatting_the_schedule

Reply:
Good suggestion - updated 🙂